app/parse.py

```diff
@@ -8,8 +16,63 @@
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s]:  %(message)s",
+    handlers=[
+        logging.FileHandler("parser.log"),
+        logging.StreamHandler(sys.stdout),
+    ],
+)
+
+
+def parse_single_quote(quote: BeautifulSoup) -> Quote:
+    return (
+        Quote(
+            text=quote.select_one(".text").text,
+            author=quote.select_one(".author").text,
+            tags=[tag.get_text() for tag in quote.select(".tags .tag")],
+        )
+    )
+
+
+def get_single_page_quotes(page_soup: BeautifulSoup) -> [Quote]:
+    quotes = page_soup.select(".quote")
+    return [parse_single_quote(quote_soup) for quote_soup in quotes]
+
+
+def get_quotes() -> [Quote]:
+    logging.info("Start parsing quotes")
+    page_num = 1
+    all_products = []
```

Comment:
Consider renaming `all_products` to `all_quotes` to better reflect the purpose of this variable, which is to store quotes.